.gitignore

```diff
@@ -67,8 +67,8 @@ Pods/*
 Carthage/Build/
 
 # Accio dependency management
-Dependencies/
-.accio/
+# Dependencies/
```

Comment:
Looks like now that you ignore entire `Pods` dir (since #66) so this is meaningless.

Reply:
@ikesyo, I ignore the entire Pods directory but I also do a `pod install` during the build step which will pull in all the pods. I can remove this and see if a simple pod install will work, but my impression was that only doing pod install will cause the build issue (https://github.com/Quick/Nimble/issues/490). I can revert this change and see if it still the case

Reply:
ah, looks like it passes without the pod deintegrate. great!

Reply:
> it passes without the pod deintegrate

Yes, that is the poiint.